src/models.ts

```diff
@@ -346,3 +346,15 @@ export interface ISelection {
   regions: IIdentityValue<IEqualsDataReference | IBetweenDataReference>[];
   filters: (IBasicFilter | IAdvancedFilter)[];
 }
+
```

Comment:
I think that Permissions should be passed in jwt token not here.
please re-think it.